versioned_docs/version-23.10/getting-started/actions.md

```diff
@@ -3,16 +3,16 @@ id: actions
 title: What actions can you take?
 ---
 
-Once you have [set up your monitoring](first-supervision.md) and resources are being successfully monitored, go to **Monitoring > Resources Status** to follow the status of resources in real time.
+Once you have [set up your monitoring](first-supervision.md) and resources are being successfully monitored, go to **Monitoring > Resources Status** to track the status of resources in real time.
 
 On this page you can perform the following actions (hover over the resource's status and click the corresponding button):
 
-- **Acknowledge a resource**: When an event occurs on a resource, you can mark the resource as acknowledged, to notify your team that you are aware of the incident and that you will take action to resolve it.
+- **Acknowledge a resource**: When an event occurs on a resource, you can mark the resource as acknowledged to notify your team that you are aware of the incident and that you will take action to resolve it.
 
-   When a resource is acknowledged, notifications are stopped, and the resource is highlighted yellow in monitoring screens.
+   When a resource is acknowledged, notifications are stopped and the resource is highlighted yellow in monitoring screens.
 
-   Acknowledging a resource does not mean that the incident is over: it will be over when the resource is back to its nominal state (OK or UP).
+   Acknowledging a resource does not mean that the incident is over. It will be over when the resource is back to its nominal state (OK or UP).
 
-- **Plan a downtime**: A downtime is a time period during which notifications are disabled for a resource. When you know that a specific resource is going to be down for maintenance, set a downtime on it to avoid getting unnecessary alerts.
+- **Plan a downtime**: Downtime is a period during which notifications are disabled for a resource. When you know that a specific resource is going to be down for maintenance, set downtime on it to avoid receiving unnecessary alerts.
```

Comment:
```suggestion
- **Plan downtime**: Downtime is a period during which notifications are disabled for a resource. When you know that a specific resource is going to be down for maintenance, set downtime on it to avoid receiving unnecessary alerts.
```